### DISCUSSION TITLE
Trouble with events getting to HA

### DISCUSSION THREAD

**Discussion by codemannh** [blakeblackshear/frigate#5270] [General]
Pretty new to Frigate and love the way it works. I have it running as an independent docker container and use MQTT to get the integration with HA to work. When I first installed it, this was not a problem, I had motion events that were reliably generated in HA. Now I'm trying to troubleshoot why most events are not getting into HA. Some do, but it woefully inconsistent. I'd like to troubleshoot myself, but I'm not sure where I should be looking. Using MQTT Explorer I can see the Frigate topics for each of my 2 cameras. Below them I can see recordings snapshot and detect along with the objects configured for each camera to detect. I thought I should see one for events, but nothing is there. Am I looking in the wrong place? 
When I look in HA, the entity for motion on both cameras exist, but only occasionally show a logbook entry. 
Ultimately I have some lights I want to turn on when motion is detected, not a specific object. The lights would be turned off when the motion clears after a period of time. I'm sure that something changed with an HA update or something, but I have not been able to figure out where the issue is.
Appreciate any help.
Thanks

**Comment by NickM-27** [comment]
What version of frigate integration and frigate are you running?

**Comment by codemannh** [comment]
Sorry, left that out. 
Frigate is running 0.10.1
Integration is running v3.0.0.

**Reply by NickM-27** [reply]
That would be your problem, you're on an old version of frigate.

**Comment by codemannh** [comment]
What is the latest stable release? I've been using whatever stable-amd64 has been providing through docker. I do an update weekly.

**Reply by NickM-27** [reply]
https://github.com/blakeblackshear/frigate/releases/tag/v0.11.1

**Comment by codemannh** [comment]
I'm definitely using the wrong label then. I'll figure that out and get the latest version installed. Thanks

**Comment by codemannh** [comment]
I got the new version installed and had some issues with ffmpeg due to hardware acceleration. I ended up turning it off and everything appears to be working.  
Thanks for your help.

**Reply by NickM-27** [reply]
feel free to create an issue so we can help with that

**Comment by codemannh** [comment]
So far the CPU utilization has been lower by nearly half than before the upgrade. Unless there is some reason to have it in place, I'm not seeing a reason to chase it down at the moment.
